sleap_roots/angle.py

```diff
@@ -0,0 +1,28 @@
+"""Get angle of each root."""
+
+import numpy as np
+
+
+def get_root_base_angle(pts: np.ndarray, base_ind=0, tip_ind=1) -> np.ndarray:
+    """Find angles for each root.
+
+    Args:
+        pts: Numpy array of points of shape (instances, nodes, 2).
+        base_ind: Index of base node in the skeleton (default: 0).
+        tip_ind: Index of distal node in the skeleton (default: 1).
+            Use 1 to get the angle to the second node.
+
+    Returns:
+        An array of shape (instances,) of angles in degrees, modulo 360.
+    """
+    while np.isnan(pts[:, tip_ind]).all() and tip_ind < int(pts.shape[1] / 2 - 1):
+        tip_ind += 1
```

Comment:
This is a cool method for finding the most distal point but maybe should be done in a separate function? It would be a good fit for the `tips` module @eberrigan started in #12. You could add this as its own function in that module after #12 is merged to `main`.